pkg/status/registry.go

```diff
@@ -14,17 +18,29 @@ type Registry interface {
 	SetGroupStatus(*statusv1.GroupStatus)
 	GetGroupStatus() *statusv1.GroupStatus
 	Child(key string) Registry
+	ChildKV(key, value string) Registry
 	ChildIfExists(key string) Registry
+	ChildIfExistsKV(key, value string) Registry
 	Parent() Registry
 	Root() Registry
 	Detach()
 	Key() string
+	Value() string
 	HasError() bool
 	GetLogger() *log.Logger
+	GetAlerter() alerts.Alerter
 }
 
 var _ Registry = &registry{}
 
+const (
+	uriKey       = "uri"
+	alertChannel = "status_registry"
```

Comment:
@DariaKunoichi,
Channels are supposed to be user provided, right? Why do we have this hardcoded? cc: @kwapik